src/modules/TeamsList/components/Teams/components/TeamItem/TeamItem.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC, useState } from 'react';
+import { TeamItemStyled, TeamItemTableWrapper } from './styled';
+import { User } from 'types';
+import { MembersListToggle } from '../MemberListToggle/MemberListToggle';
```

Comment:
name file index.tsx and rename import to import { MembersListToggle } from '../MemberListToggle';

Reply:
done